src/main/webapp/index.html

```diff
@@ -43,6 +43,9 @@
     <title>smallcity</title>
   </head>
   <body>
+    <div id="loader-container">
+      <div id="loader"></div>
```

Comment:
Caveat: this is more of a preference, than anything else. Reading this, and the method like "initiateLoader" I'd expect code for some process that actually loads or kickstarts something. This is is a ui indicator that doesn't do this. If everyone on your team is ok with it, it's fine. If not, perhaps a name that indicates it's more of a progress bar ordeal. Your comment below calls this a 'loader circle'. I find that a bit clearer.

Reply:
I changed all of the names to loader-circle or loaderCircle